pkg/storage/s3.go

```diff
@@ -0,0 +1,49 @@
+package storage
+
+import (
+	"github.com/pingcap/errors"
+	"github.com/pingcap/kvproto/pkg/backup"
+	"github.com/spf13/pflag"
+)
+
+const (
+	s3EndpointOption = "s3.endpoint"
+)
+
+type S3BackendOptions struct {
+	Endpoint string `json:"endpoint" toml:"endpoint"`
+}
+
+func (options *S3BackendOptions) validate() error {
+	return nil
+}
+
+func (options *S3BackendOptions) isEmpty() bool {
+	return options.Endpoint == ""
+}
```

Comment:
Can we combine these two functions?
If we have 2 options for S3, both of them have default values, how can we determine if it's empty?

Reply:
Removed these two functions.